src/qgitrebase.cpp

```diff
@@ -26,15 +26,14 @@ namespace LibQGit2
 {
 
 struct Rebase::Private {
-    Private(git_rebase *rebase, const RebaseOptions &opts)
-        : mRebase(rebase, git_rebase_free),
-          mOpts(opts)
+    Private(git_rebase *rebase)
```

Comment:
The `qgitrebaseoptions.h` and `qgitcheckoutoptions.h` includes can be removed in this file.

Reply:
Removed.